test/WebSites/RazorWebSite/EmbeddedViews/Views/EmbeddedHome/EmbeddedPartial.cshtml

```diff
@@ -0,0 +1 @@
+Hello from EmbeddedHome\EmbeddedPartial
```

Comment:
Intentionally not including new lines to avoid dealing with in the test. Let me know if you feel strongly about it.